python/GafferUI/CompoundEditor.py

```diff
@@ -124,6 +124,49 @@ def restoreWindowState( self ) :
 		for p in self.__detachedPanels :
 			p.restoreWindowState()
 
+	# Returns the editor of the specified type that has the mouse focus. This
```

Comment:
It's keyboard rather than mouse focus isn't it? I don't see anything in this function that considers mouse position...

Reply:
Based on our off-line chat, renamed this method in https://github.com/GafferHQ/gaffer/pull/3372/commits/5129e4597c2dc9c844e6e05e387032c3e65588fc